rootfs/opt/kubectl/example/GoogleBackup/configmap/restore.sh

```diff
@@ -0,0 +1,21 @@
+set -e
+set -x
+
+echo "Start restore"
+
+gcloud auth activate-service-account --key-file=/secret-backup/neural-theory-215601-53bf50004612.json
+
+gsutil -m rsync -P -r gs://qserv-backup/qserv-dev/"$HOSTNAME" /qserv/data 
+
+# Google bucket do not preserve uid/gid for directories
+chown -R 1000:1000 /qserv/data
+
+STATUS_FILE="/tmp/$HOSTNAME-restore-ok"
+touch "$STATUS_FILE"
+gsutil -m cp -r "$STATUS_FILE" gs://qserv-backup/qserv-dev/results/
```

Comment:
This procedure is a prototype which was aimed at being ran only one time, so I didn't polish all the very details... As backup/restore pods do not exit on success, I found this solution for knowing if backup is ok. Of course you need to clear results/ each time you create new PVCs. The final goal would be to use job for backup for job do not support PVCtemplate so all the mappin between backup/restore Jobs and PVC would have to be done by hand, this would be very cost effective. FYI, this backup/procedure is provided as example and will not be maintain (but i may be re-used to perform manual backup or to write a production ready backup in the long-term)